src/components/grid-aware/ProgramBlock/ProgramBlock.module.css

```diff
@@ -0,0 +1,139 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: var(--bleed-grid-template-columns);
+  margin: auto;
+  margin-bottom: 100px;
+  max-width: var(--bleed-max-width);
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.bleedWrapper > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+    margin-bottom: 80px;
+  }
+}
+
+.bleedMainContent {
+  grid-column: 2;
+}
+
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+}
+
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: flex;
+    flex-direction: column-reverse;
+  }
+}
+
+
+/* Main grid areas */
+
+.gridAreaText {
+  grid-column: 6/ span 6;
+  grid-row: 1;
+}
+
+@media (--tablet-and-down) {
+  .gridAreaText {
+    padding-top: 16px;
+  }
+}
+
+.gridAreaImage {
+  grid-column: 2 / span 4;
+  grid-row: 1;
+}
+
+/* Left grid area */
+
+.title {
+  font: var(--font-body-large);
+  margin-bottom: 5px;
+  margin-top: 0;
+}
+
+@media (--tablet-and-down) {
+  .title {
+    margin-top: 0;
+  }
+}
+
+.subtitle {
+  font: var(--font-subtitle);
+}
+
+.description {
+  font: var(--font-body-medium);
+  margin-top: 20px;
+}
+
+.description :first-child {
```

Comment:
You should probably use the direct child selector, `>`, here so that you don't select all first children in the whole tree. For example, not only does this select the first child of `.description`, but if you have a `<ul><li>Item 1</li><li>Item 2</li></ul>`, then it'll also select that first `<li>` because it's the first child of `<ul>`.